setup.cfg

```diff
@@ -80,6 +80,11 @@ all =
     %(doc)s
     %(test)s
 
+
+[options.entry_points]
+console_scripts =
+    templateflow_get=templateflow.templateflow_get:main
```

Comment:
```suggestion
    templateflow=templateflow.templateflow_get:main
```

Reply:
hum I'm affraid it will collapse will templateflow name space....

Reply:
have you tried? because it won't...